### DISCUSSION TITLE
[Wiki] Add a link to Config file page on the Installation procedures

### DISCUSSION THREAD

**Discussion by igorsantos07** [asmvik/yabai#2459] [Ideas]
At the end of the [install page](https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#configure-scripting-addition), there's a request to edit the as-of non-existing config file. I got a bit puzzled, but gladly there was a link to it on the sidebar. I would suggest linking to it directly from the text, further explaining the file may not exist and would need to be created, as described in that other page: https://github.com/koekeishiya/yabai/wiki/Configuration#configuration-file

**Comment by asmvik** [comment]
Done.
